Applications/Scale/test/testScale.cpp

```diff
@@ -184,3 +185,45 @@ void scaleGait2354_GUI(bool useMarkerPlacement)
 
     delete subject;
 }
+
+void scaleModelWithLigament()
+{
+    // SET OUTPUT FORMATTING
+    IO::SetDigitsPad(4);
+
+    std::string setupFilePath("");
+    ScaleTool* subject;
+    Model* model;
+
+    // Remove old model if any
+    FILE* file2Remove = IO::OpenFile(setupFilePath + "toyLigamentModelScaled.osim", "w");
+    fclose(file2Remove);
+
+    // Construct model and read parameters file
+    subject = new ScaleTool("toyLigamentModel_Setup_Scale.xml");
+
+    // Keep track of the folder containing setup file, wil be used to locate results to comapre against
+    setupFilePath = subject->getPathToSubject();
+
+    model = subject->createModel();
+
+    SimTK::State& s = model->updWorkingState();
+    model->getMultibodySystem().realize(s, SimTK::Stage::Position);
+
+    if (!model) {
+        //throw Exception("scale: ERROR- No model specified.",__FILE__,__LINE__);
```

Comment:
This check should be moved up to after asking the scale to create a model. Too late here when you've already used it.